tools/daemon/flow.c

```diff
@@ -631,6 +642,7 @@ static inline void get_htid(struct flow_ctx *ctx, int prio, int *ht_krn, int *ht
 		}
 	}
 
+	free_pending_list(ctx);
```

Comment:
please consider more suitable place to call this. May be directly from del_flow()/ add_flow() to be actual and avoid memory leak during cleanup.